CHANGES

```diff
@@ -7,6 +7,7 @@ Changelog
  * removed the Swift_Transport_MailInvoker interface and Swift_Transport_SimpleMailInvoker class
  * removed the Swift_SignedMessage class
  * removed newInstance() methods everywhere
+ * methods operating on Date header now use DateTimeImmutable object instead of Unix timestamp
```

Comment:
I have added a mention of the renaming from getTimestamp()/setTimestamp() to getDateTime()/setDateTime().